operators/advanced_network/advanced_network/CMakeLists.txt

```diff
@@ -0,0 +1,92 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+cmake_minimum_required(VERSION 3.20)
+project(advanced_network)
+
+find_package(holoscan 0.6 REQUIRED CONFIG
```

Comment:
Have we confirmed 0.6 compatibility with ANO?

Reply:
Ty, just moved that, but that seems unlikely. @cliffburdick any recent versions you know we need? Last time I wrote the debian package I listed 2.6 for apt.

Reply:
I've only been testing on the newest versions, so we definitely should bring this up

Reply:
upped to 2.6 (last time I had tested) for ano

Reply:
Unresolved because GitHub still shows 0.6 for this line...

Reply:
my bad, I updated basic_network instead, ty for catching